.github/workflows/generate-artifacts.yml

```diff
@@ -0,0 +1,69 @@
+name: allcontributors-zenodo
+
+on:
+  pull_request: []
```

Comment:
with above comment in mind -- shouldn't it be ran on branches or at least `master` by default as well?

Reply:
We would want any changes to the action to be tested first.

Reply:
correct, but also react on the final state (which might have some others already added etc) in master, right?